src/consumer/kafka.rs

```diff
@@ -42,11 +44,20 @@ use tracing::{debug, error, info, instrument};
 pub async fn start_consumer(
     topics: &[&str],
     kafka_client_config: &ClientConfig,
+    default_topic_partitions: i32,
     spawn_actors: impl FnMut(
         Arc<StreamConsumer<KafkaContext>>,
         &BTreeSet<(String, i32)>,
     ) -> ActorHandles,
 ) -> Result<(), Error> {
+    create_missing_topics(
+        kafka_client_config.clone(),
+        topics,
+        default_topic_partitions,
+    )
+    .await
+    .expect("Failed to find/create topics");
```

Comment:
Would we want to create/check for topics in production?

Reply:
Why not? It only happens on startup and will no-op.